Tasks/Savanchuk/Task7/task7_5.py

```diff
@@ -0,0 +1,40 @@
+#5. Write a decorator function that caches the 
+#return value of a function with a dictionary. 
+#If the function is called again with the same arguments, 
+#return the cached value instead of computing it again.
+def cashing(func):
+    def wrapper(*args):
+        dict = {}
```

Comment:
this dict will be recreated for each new function call, the meaning of cache is kinda lost here, it should be placed outside of a wrapper